test/module/child_process/basics.js

```diff
@@ -0,0 +1,30 @@
+// Test basic child process functionality
+//
+
+async_test(function(){
+    var text = "hello";
+    var process = require('child_process');
+    var p = process.spawn("echo", ["-n", text]);
```

Comment:
Command `echo` doesn't need argument `-n` on Windows. Otherwise the output will be `-n hello`